pkg/deploy/generator/resources.go

```diff
@@ -755,7 +760,7 @@ az account set -s "$SUBSCRIPTIONID"
 systemctl start docker.service
 az acr login --name "$(sed -e 's|.*/||' <<<"$ACRRESOURCEID")"
 
-MDMIMAGE="${RPIMAGE%%/*}/genevamdm:master_41"
+MDMIMAGE="${RPIMAGE%%/*}/${MDMIMAGE##*/}"
```

Comment:
Took me a few moments to understand the pattern manipulation here. Is it possible to just use the full image URI directly?

Reply:
No, otherwise we would.  This is a hack which is saving an additional configuration entry in the RP config repo.
The registry prefix depends on whether we are running in int or prod - this hack is ensuring that the correct registry is used to pull the geneva image.